src/components/Contact.js

```diff
@@ -12,6 +13,42 @@ const styles = (theme) => ({
 });
 
 class Contact extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      message: '',
+    };
+  }
+
+  handleEmailChange = (event) => {
```

Comment:
There is a more efficient way to do this for forms with multiple inputs.
https://hjnilsson.com/2016/12/11/generic-form-handleChange-in-react/

There's lots of ways you can tackle it but you don't have to change it now